### PR TITLE
Fix error where multipolygon features have invalid coordinates

### LINKED ISSUE #39: MultiPolygon type for a Point feature breaks geojsonhint
An error is thrown when `MultiPolygon` type is provided to a Point feature.

`Uncaught TypeError: coords.forEach is not a function`

```
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "MultiPolygon",
        "coordinates": [
          -0.087890625,
          51.56341232867588
        ]
      }
    }
  ]
}
```

### PR DESCRIPTION
**Pull request by mcwhittemore**
Fixes #39

---

While this fixes #39 it does so by duplicating an error message to a different part of the error logic stack. The long block of if statements are a bit hard to read, so I think we should refactor those a bit too before merging this.

Before doing that, does this look right @tmcw?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (tmcw) [inline .gitignore]: These should be in a global npmignore, not local

File: object.js [hunk 2]
```diff
@@ -143,9 +143,17 @@ function hint(gj, options) {
                     });
                 }
             }
-            coords.forEach(function(c) {
-                positionArray(c, type, depth - 1, c.__line__ || line);
-            });
+            else if (!Array.isArray(coords)) {
```
Comment (tmcw) [inline object.js]: plz follow existing code style and `} else if {` instead of a linebreak between `}` and `else if`

File: .gitignore (hunk 1 above)
Comment (mcwhittemore) [inline .gitignore]: this is a gitignore not an npmignore

Comment (tmcw) [inline .gitignore]: sorry, global gitignore. had npmignores on the mind. This stuff should be in a global gitignore.

**Comment by mcwhittemore** [comment]
@tmcw comments fixed. Any thoughts on refactoring this if else block vs leaving in the same error message twice?

**Pull request merged**